File: add/codesnippet/VisualBasic/t-system.drawing.rotatef_1.vb
    Dim bitmap1 As Bitmap

    Private Sub InitializeBitmap()
        Try
            bitmap1 = CType(Bitmap.FromFile("C:\Documents and Settings\All Users\" _
                & "Documents\My Music\music.bmp"), Bitmap)
            PictureBox1.SizeMode = PictureBoxSizeMode.AutoSize
            PictureBox1.Image = bitmap1
        Catch ex As System.IO.FileNotFoundException
            MessageBox.Show("There was an error. Check the path to the bitmap.")
        End Try


    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, _
        ByVal e As System.EventArgs) Handles Button1.Click

        If bitmap1 IsNot Nothing Then
            bitmap1.RotateFlip(RotateFlipType.Rotate180FlipY)
            PictureBox1.Image = bitmap1
        End If

    End Sub

File: add/codesnippet/CSharp/t-system.drawing.rotatef_1.cs
    Bitmap bitmap1;

    private void InitializeBitmap()
    {
        try
        {
            bitmap1 = (Bitmap)Bitmap.FromFile(@"C:\Documents and Settings\" + 
                @"All Users\Documents\My Music\music.bmp");
            PictureBox1.SizeMode = PictureBoxSizeMode.AutoSize;
            PictureBox1.Image = bitmap1;
        }
        catch(System.IO.FileNotFoundException)
        {
            MessageBox.Show("There was an error." + 
                "Check the path to the bitmap.");
        }


    }

    private void Button1_Click(System.Object sender, System.EventArgs e)
    {

        if (bitmap1 != null)
        {
            bitmap1.RotateFlip(RotateFlipType.Rotate180FlipY);
            PictureBox1.Image = bitmap1;
        }

    }